bundles/framework/featuredata2/plugin/MapSelectionPlugin.ol3.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * @class Oskari.mapframework.bundle.metadata.plugin.MapSelectionPlugin
+ *
+ * Provides functionality to draw a selection box on the map
+ */
+Oskari.clazz.define(
+    'Oskari.mapframework.bundle.featuredata2.plugin.MapSelectionPlugin',
```

Comment:
Seems this doesn't need to be plugin anymore. Are there any references to mapmodule or mapping lib specific code? I think there's references to modifyControls and drawlayers that doesn't appear to be initialized anywhere. Also seems to include lots of unused code.